src/utils/src/backend-supports/project-supports.ts

```diff
@@ -3,13 +3,24 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+/**
+ * This function calculates the end date.
+ * @param start the start date
+ * @param weeks number of weeks
+ * @returns the start date after the weeks have passed
+ */
 const calculateEndDate = (start: Date, weeks: number) => {
   const end = new Date(start);
   end.setDate(start.getDate() + weeks * 7);
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+/**
+ * This symbol function calculates the duration 
+ * @param wps an array of a project schedule
```

Comment:
this is an array of work packages

Reply:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/94508024/155404865-65ef42da-494d-472d-a3b6-2edadb35af59.png">
if we change it array of Array<WorkPackage>, it would fail the pre-written tests in src/utils/tests/projects-supports.test.ts, change anyways?

Reply:
I just meant to change the comment to say 'wps is an array of work packages', and yeah good point just leave it as 'any' type in the arguments for now.